manual_flagging/flagging.py

```diff
@@ -222,14 +216,15 @@ def create_bins_and_labels(input_list):
 
 dfs_to_rolling_window = {}  # Dictionary to store DataFrames
 
-# Collect tris where our contemporary flagging methods are used
-current_tris = [tri for tri, method in tri_stat_groups.items() if method == "current"]
-
-for tri in current_tris:
+for tri in inputs["run_tri"]:
     # Iterate over housing types defined in yaml
-    for housing_type in inputs["housing_market_type"]:
+    for housing_type in [
+        ht
+        for ht in inputs["housing_market_type"]
+        if ht in inputs["stat_groups"][f"tri{tri}"]
+    ]:
```

Comment:
This sub-setting enables a subtle functionality for these yaml inputs:

```yaml
housing_market_type: ["res_single_family", "res_multi_family", "res_all"]
run_tri: [1, 2]
```

This list comprehension makes it so that data frames (on which flagging is performed later in the script) are only created ifn the `housing_market_type` is defined as a subkey under the `tri$n` key in the `stat_groups` yaml object. This is what makes it possible to run flagging over all res properties vs only single family or multi-family.